<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/imgs/64-647021_light-colour-background-hd.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="tasksLayout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="toolbarLayout">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(200, 0, 0);
border-width:thick;
background-color:rgb(222, 170, 255);
</string>
        </property>
        <property name="text">
         <string>Status: 0 todo/0 done</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addTaskButton">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(200, 0, 0);
border-width:thick;
background-color:rgb(222, 170, 255);</string>
        </property>
        <property name="text">
         <string>Add task</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="aboutusbutton">
      <property name="font">
       <font>
        <family>Microsoft JhengHei UI Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(200, 0, 0);
border-width:thick;
background-color:rgb(222, 170, 255);</string>
      </property>
      <property name="text">
       <string>A   B    O    U    T             U    S</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
